src/Admin/CommentsGridFieldApproveAction.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace SilverStripe\Comments\Admin;
+
+use SilverStripe\Comments\Model\Comment;
+use SilverStripe\Control\Controller;
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\GridField\GridField_ActionMenuItem;
+use SilverStripe\Forms\GridField\GridField_ActionProvider;
+use SilverStripe\Forms\GridField\GridField_ColumnProvider;
+use SilverStripe\Forms\GridField\GridField_FormAction;
+
+class CommentsGridFieldApproveAction implements
+    GridField_ColumnProvider,
+    GridField_ActionProvider,
+    GridField_ActionMenuItem
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function augmentColumns($gridField, &$columns)
+    {
+        if (!in_array('Actions', $columns)) {
+            $columns[] = 'Actions';
+        }
+    }
+
+    public function getTitle($gridField, $record, $columnName)
+    {
+        return _t(__CLASS__ . '.APPROVE', 'Approve');
+    }
+
+    public function getExtraData($gridField, $record, $columnName)
+    {
+
```

Comment:
Is PHPCS not properly configured on this repo?

Reply:
I don't understand this comment sorry

Reply:
There's a blank line here. Normally you get a "Blocks should not be padded with empty lines" or something.

Reply:
Ah. Perhaps the config differs from our normal, but doesn't look like it. 🤔 strange 😕

Reply:
I could find only a single discrepancy between e.g. Elemental and this module - `<exclude name="PSR1.Files.SideEffects.FoundWithSymbols" />` - I don't know why this isn't being picked up :/

Reply:
All good =)

Reply:
In any case I've removed the space at the top of this method, and added a space between this method and the next :>